README.md

```diff
@@ -17,6 +17,11 @@ You can then enable the serverextension
 
     jupyter serverextension enable --py nbgitpuller --sys-prefix
 
+# How to use `nbgitpuller`
+
+[Follow the `nbgitpuller` usage guide](https://the-littlest-jupyterhub.readthedocs.io/en/latest/howto/content/nbgitpuller.html)
```

Comment:
I was about to suggest the use of tljh.jupyter.org instead of the full written out name, but...

```
curl -L https://zero-to-jupyterhub.readthedocs.io/en/latest # WORKS
curl -L https://z2jh.jupyter.org # WORKS IDENTICALLY AS ABOVE
curl -L https://the-littelest-jupyterhub.readthedocs.io/en/latest # WORKS
curl -L https://tljh.jupyter.org/en/latest # DOES NOT WORK AND OUTPUT:
# curl: (51) SSL: no alternative certificate subject name matches target host name 'tljh.jupyter.org'

Reply:
Note that visiting https://tljh.jupyter.org is okay, but visiting https://tljh.jupyter.org/en/latest isn't, while the same for z2jh is fine no matter what (https://z2jh.jupyter.org / https://z2jh.jupyter.org/en/latest)

Reply:
oh no! I wonder what happened with the SSL propagation. both tljh and z2jh point to a readthedocs implementation, so maybe it's in the cloudflare DNS setup? I can't access the account because I'm in Europe so it doesn't recognize the IP address. Maybe @Carreau can take a look? I think they're the only ones w/ access

Reply:
`/me` got +50 xp by being exposed to a new concept: *SSL propagation*

Only 500 xp left until level up :D

Reply:
`/me` may or may not have totally made up the word SSL propagation and if it's a real thing that might be a total coincidence :-)

Reply:
`tljh.jupyter.org` is not a thing in cloud flare, and we ran out of free rules to redirect domains.

It's $1/rule/month if the financial board is ok buying or we could remove some old rules.

Reply:
@choldgraf I also delegated one account to you so you can access access-token.

Reply:
So it seem that there is something we can do as tljh.jupyter.org is and alias to the-littlest-jupyterhub.readthedocs.io but the configuration is govern by RTD, so someone will need to give one of us access to the RTD admin pannel. @yuvipanda  ?

Reply:
@Carreau can you create a RTD account? Then somebody can add you (I just tried but it said user "Carreau" did not exist)